elements/pfe-badge/src/pfe-badge.html

```diff
@@ -0,0 +1,3 @@
+<span class="pfe-badge__container">
+    <slot></slot>
```

Comment:
I think we should consider getting rid of the slot and make this component more like pfe-number.
https://github.com/patternfly/patternfly-elements/blob/master/elements/pfe-number/src/pfe-number.html

I think the approach that we need to take is to take the value from the `number` or `text` attribute, put it as the `textContent` of the span in `pfe-badge.html` and then update the lightDOM with `this.textContent`.

Reply:
@kylebuch8 hey I updated the code and now using number or text attribute value to set textcontent of the span